test/org/sagebionetworks/bridge/services/backfill/BackfillRecordFactoryTest.java

```diff
@@ -0,0 +1,58 @@
+package org.sagebionetworks.bridge.services.backfill;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+import org.sagebionetworks.bridge.models.BackfillRecord;
+import org.sagebionetworks.bridge.models.BackfillTask;
+import org.sagebionetworks.bridge.models.studies.Study;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.stormpath.sdk.account.Account;
+
+public class BackfillRecordFactoryTest {
```

Comment:
For completeness, should we also test createAndSave(), if if only to make sure it calls through to a mock dao?